Wallet Guide on Placing Orders (#1096)
TBD54566975/developer.tbd.website: site/docs/tbdex/wallet/send-rfq.mdx

At this point, the RFQ has been successfully sent to the PFI and an **exchange** has been created.
This exchange serves as a thread of messages between your application and the PFI regarding this transaction.

> Your application will poll this exchange until a `Quote` message appears in the exchange:

Author: ```suggestion
Your application will poll this exchange until a message of kind `quote` appears in the exchange:
```

    filter: { id: rfq.exchangeId }
  });

>   const [ exchange ] = exchanges.data;

Author: ```suggestion
  const exchange = exchanges.data[0];
```

>   const [ exchange ] = exchanges.data;

Author: ```suggestion
  quote = exchange.find(msg => msg.kind === 'quote');
```

  });

  const exchange = exchanges.data[0];

>   quote = exchange.find(msg => msg instanceof Quote);

Author: ```suggestion
  quote = exchange.find(msg => msg.kind === 'quote');
```

}
```

> Now that you've received a Quote, you can present this to your customer and give them the option to [place an Order](/docs/tbdex/wallet/place-order).

Author: ```suggestion
Now that you've received a Quote, you can present this to your user and give them the option to place an Order.
```
